### DISCUSSION TITLE
EddieHub Guild Video Call for MLH Init 2022

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1915] [Ideas]
Hey folks, What are your thoughts on conducting a video call with all our guild members via zoom or google meet. We can focus on discussing things, motivate and appreciate all the folks completing tasks and helping our guild to climb up the leader board. We would appreciate your thoughts on this.
- Day of conduction
- Time to be conducted
- Platform you would recommend
- Any more ideas to make it fun

**Comment by TripathiShyaamal** [comment]
Day- 30th June
Time- 7 to 9 PM
Platform -Google meet

**Comment by srisatya12** [comment]
Date - 2nd July
Timings - 5:00pm - 7:00pm
Platform - Google Meet

**Comment by balayokesh** [comment]
Day- Any weekend days
Time- After 6pm IST (evening)
Platform - Google meet
